### DISCUSSION TITLE
Why does renovate ignore semVer refs in ocirepository?

### DISCUSSION THREAD

**Discussion by horihel** [renovatebot/renovate#31684] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure Devops, Renovate 37.440.7

### Please tell us more about your question or problem

I just found out that renovate only supports "tag" and "digest" refs in the flux manager. 
Some teams used "[semver](https://fluxcd.io/flux/components/source/ocirepositories/#semver-example)" ref and flux is ignoring them as "unversioned-reference".

Is this just a missing feature or is there a specific reason why semver is not supported by renovate in this context?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Missing feature

**Reply by HonkingGoose** [reply]
> I just found out that renovate only supports "tag" and "digest" refs in the flux manager.

I created this issue, to add support for tags and digests to the `flux` manager. But you say Renovate already supports tag and digest refs...

I'll let a maintainer check if we still need this issue:

- https://github.com/renovatebot/renovate/issues/29768

**Reply by rarkins** [reply]
I think it's still valid. semver needs a new feature request
